app/views/layouts/customer_layout.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BackendValidations</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <%= yield %>
+  </body>
+  <% if !flash[:alert].nil? %>
+    <p class="my_alert"><%= flash[:alert] %></p>
+<% end %>
```

Comment:
This should be inside of the **body** tag. It is causing errors to appear in different pages e.g. customers/new.